spec/regression/GH-1106_fiber_must_propagate_java_exceptions_spec.rb

```diff
@@ -38,7 +38,7 @@
       
       Thread.pass while t.status
       
-      expect(t.status).to eq(false) # dead
+      expect(t.status).to eq(nil) # terminated with an exception
```

Comment:
```
Thread.new { Thread.exit }.join.status => false
```
there's an internal exception but it shouldn't be propagated

```suggestion
      expect(t.status).to eq(false) # dead
```